splash/casperjs_utils.py

```diff
@@ -0,0 +1,114 @@
+# Copyright (c) 2011-2015 Nicolas Perriault
```

Comment:
could you please add a comment with a link to the casperjs file with relevant source code?